### DISCUSSION TITLE
Patches Listing

### DISCUSSION THREAD

**Discussion by Noorilahi** [amzn/selling-partner-api-models#3209] [General]
Order Api, Finance Api and Report Api are Working fine but when i try for Patche a listing using Listing Api it is showing error.

Function Call:
file = {
            "productType": "PRODUCT",
            "patches": [
                {
                    "op": "replace",
                    "path": "/attributes/fulfillment_availability",
                    "value": [
                        {
                            #"fulfillment_channel_code": "DEFAULT",
                            "quantity": 15,
                        }
                    ]
                }
            ]
        }
res = ListingsItems(credentials=self.credentials).patch_listings_item(
                sellerId=seller_id,
                sku="B09DPCV6PF",
                marketplaceIds=mark,
                issueLocale="en_IN",
                body=file,
            )

ERROR "

sp_api.base.exceptions.SellingApiForbiddenException: [{'message': 'Access to requested resource is denied.', 'code': 'Unauthorized', 'details': ''}]

**Comment by TechyChap** [comment]
When you created your App registration in Seller Central / Vendor Central did you ensure you gave permission to the listings api?
